packages/ses/src/lockdown-shim.js

```diff
@@ -57,6 +59,37 @@ import { makeCompartmentConstructor } from './compartment-shim.js';
 
 const { Fail, details: d, quote: q } = assert;
 
+// eslint-disable-next-line no-restricted-globals
+const TO_PRIMITIVE_SYM = Symbol.toPrimitive;
+
+// eslint-disable-next-line no-extend-native, no-restricted-globals
+Object.prototype[TO_PRIMITIVE_SYM] = {
+  [TO_PRIMITIVE_SYM](hint) {
+    if (
+      typeof this === 'object' &&
+      this &&
+      getPrototypeOf(this) === objectPrototype &&
+      hint === 'number'
+    ) {
+      // eslint-disable-next-line @endo/no-polymorphic-call
+      Fail`Suppressing conversion of ${q(this.toString())} to number`;
```

Comment:
Slightly worried about an own `toString()` that would recursively try trigger `@@toPrimitive`.
If the intent is to used a polymorphic call, why not add a reentrancy guard around the whole method?

```suggestion
      Fail`Suppressing conversion of ${q(Object.prototype.toString.call(this))} to number`;
```